src/main/resources/locales/zh-cn.yml

```diff
@@ -1,33 +1,40 @@
-homes_menu_title: '%1%''的家'
+homes_menu_title: '%1% 的家'
 public_homes_menu_title: '开放的家'
 warps_menu_title: '地标'
+home_editor_title: '编辑家: %1%'
+warp_editor_title: '编辑地标: %1%'
 item_name: '[%1%](#00fb9a)'
 item_description: '&7ℹ %1%'
 item_description_blank: ''
 item_controls_space: ''
-item_controls_left_click: '[左键点击:](#00fb9a) [传送](gray)'
-item_controls_right_click: '[右键点击:](#00fb9a) [编辑](gray)'
-item_controls_drag_drop_item: '[Drag+Drop Item:](#00fb9a) [Set icon](gray)'
+item_controls: '[  • 左键点击: 传送](gray)\n[  • 右键点击: 编辑](gray)\n[  • 放置物品: 设置图标](gray)\n'
+item_deleted_name: '&c[已删除] [%1%](#00fb9a)'
 home_owner_name: '[[由 %1% 创建]](#00fb9a)'
-pagination_first_page: '[⏪ 首页 \\(1\\)](#00fb9a)'
-pagination_previous_page: '[◀ 上一页 \\(%prevpage%\\)](#00fb9a)'
-pagination_next_page: '[下一页 \\(%nextpage%\\) ▶](#00fb9a)'
-pagination_last_page: '[尾页 \\(%pages%\\) ⏩](#00fb9a)'
-menu_controls_title: '[说明](gray)'
-menu_controls_details: '[  • 左键点击: 传送](gray)\n[  • 右键点击: 编辑](gray)\n[  • 放置物品: 设置图标](gray)\n'
-home_editor_title: '编辑家: %1%'
-warp_editor_title: '编辑地标: %1%'
-back_button: '[返回](#00fb9a)'
 edit_location_button: '[更新位置](#00fb9a)'
-edit_name_button: '[修改名称](#00fb9a)'
-edit_description_button: '[修改描述](#00fb9a)'
+edit_location_default_message: '[坐标:](#00fb9a) x: %1%, y: %2%, z: %3%'
 edit_name_title: '修改名称: %1%'
+edit_name_button: '[修改名称](#00fb9a)'
 edit_description_title: '修改描述: %1%'
-edit_privacy_button: '[隐私设置](#00fb9a)'
+edit_description_button: '[修改描述](#00fb9a)'
+edit_description_default_message: '&7ℹ %1%'
+edit_description_default_message_blank: ''
+edit_description_default_input: '没有描述' # Leave blank '' to fill in the icon item name, enter a blank ' ' to empty the description
+edit_privacy_button: '[切换开放状态](#00fb9a)'
+edit_privacy_message: '[当前为](#00fb9a) %1% [状态](#00fb9a)'
+edit_privacy_message_var1_public: '&a开放'
+edit_privacy_message_var1_private: '&6私有'
```

Comment:
Could we have a better name for this locale key? Kind of weird to have "var1" and "var2" in it, bit confusing. Also, comment on `edit_description_default_input` should be removed -- avoid comments in this file